src/core/dyad_core.c

```diff
@@ -69,12 +83,11 @@ static int gen_path_key (const char* str,
 }
 
 #if DYAD_PERFFLOW
-__attribute__ ((annotate ("@critical_path()")))
-static dyad_rc_t dyad_kvs_commit (const dyad_ctx_t* ctx,
-                                  flux_kvs_txn_t* txn)
+__attribute__ ((annotate ("@critical_path()"))) static dyad_rc_t dyad_kvs_commit (
```

Comment:
This is actually where a new line is needed.

Reply:
This is a clang format thing. I'll fix it.